pandas/core/groupby/generic.py

```diff
@@ -143,8 +143,12 @@ def _cython_agg_blocks(self, how, alt=None, numeric_only=True, min_count=-1):
         new_blocks = []
         new_items = []
         deleted_items = []
+        no_result = object()
```

Comment:
why don't you just set ``result = None`` outside the try instead as its more clear

Reply:
because I don't know for sure that `None` won't be the correct `result` produced on line 154